src/ontology/mondo-edit.obo

```diff
@@ -584370,6 +584370,17 @@ is_a: MONDO:1030005 {source="PMID:31120550"} ! mosaic SMO syndrome
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7768" xsd:anyURI
 
+[Term]
+id: MONDO:1030008
+name: mitral valve insufficiency
+def: "Dysfunction of the mitral valve characterized by incomplete valve closure." [https://orcid.org/0009-0009-6791-365X, NCIT:C50888]
+synonym: "mitral insufficiency" EXACT [NCIT:C50888]
+synonym: "mitral valve insufficiency/ regurgitation" EXACT [NCIT:C50888]
+xref: NCIT:C50888 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003767 {source="https://orcid.org/0009-0009-6791-365X"} ! mitral valve disorder
```

Comment:
it looks like you can also use NCIT as a source for this subclass Of (according to https://www.ebi.ac.uk/ols4/ontologies/ncit/classes?short_form=NCIT_C50888)

Reply:
added